database-cassandra/seeding/seeding.js

```diff
@@ -1,45 +1,32 @@
 const faker = require('faker');
 const path = require('path');
-const restaurantsList = require('../restaurantList.js.js');
-const photosList = require('../photosList.js.js');
+const restaurantsList = require('../restaurantList.js');
+const photosList = require('../photosList.js');
 const restaurants = restaurantsList.restaurants;
 const maxphotosPerReview = 3;
-const numUsers = 2;
-const numRestaurants = 2; // leave equal to num users otherwise might break some function below
-const numReviews = 2;
+const numUsers = 1000000;
+const numRestaurants = 1000000; // leave equal to num users otherwise might break some function below
+const numReviews = 1000000;
+const maxQuery = 300;
 const createCsvWriter = require('csv-writer').createObjectCsvWriter;
 
 const cassandra = require('cassandra-driver'); 
 const authProvider = new cassandra.auth.PlainTextAuthProvider('root', 'AKK');
 const contactPoints = ['127.0.0.1:9042'];
 const keyspace = 'yelpreviews';
-const client = new cassandra.Client({contactPoints: contactPoints, authProvider: authProvider, keyspace: keyspace});
-
-//Ensure all queries are executed before exit
-
+const client = new cassandra.Client({contactPoints: contactPoints, authProvider: authProvider, keyspace: keyspace, localDataCenter: 'datacenter1'});
+const { spawn } = require('child_process');
 
 const writeToCsv = (header, records, filename) => {
   let csvWriter = createCsvWriter({
-    path: `database/postgres-seeding/csv/${filename}.csv`,
+    path: `database-cassandra/seeding/csv/${filename}.csv`,
     header: header
   });
   console.log(`Writing ${filename}`);
   return csvWriter.writeRecords(records);
 }
 
 let db = {
-  execute: function execute(query, params, callback) {
-    return new Promise((resolve, reject) => {
-      client.execute(query, params, (err, result) => {
-        if(err) {
-          reject()
-        } else {
-          callback(err, result);
-          resolve()
-        }
-      });
-    });
-  },
   querySQL: function (query, successLog = 'Success') {
```

Comment:
I don't understand the purpose of this function in the cassandra seed file, using pool queries made sense in the psotgres seed file, but here I am unsure.